lib/dry/auto_inject/dependency_map.rb

```diff
@@ -33,7 +35,7 @@ def names
       def to_h
         @map.dup.to_h
       end
-      alias_method :to_hash, :to_h
+      alias to_hash to_h
```

Comment:
I thought it was better to use `alias_method` than `alias` https://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html

Reply:
Hm, it's a good idea for a new spec. I missed that, thanks!

Reply:
Yeah, I'm not a fan of the lexical scoping of `alias`. Let's leave it as `alias_method`.